docs/splitter.css

```diff
@@ -0,0 +1,37 @@
+html, body, #app
+{
+    height: 100%;
+    margin: 0;
+}
+
+#split-view {
+  height: 100%;
+  display: flex;
+  align-items: stretch;
+  flex-direction: row;
+}
+
+.sidebar {
+  width: 50%;
+  margin: 0;
+  order: 1;
+  border-left: 2px solid #dde1e4;
+  background-color: #fff;
+  overflow: hidden;
+}
+
+.main-view {
+  width: 50%;
+  order: 0;
+  display:flex;
+  align-items: stretch;
+  flex-direction: column;
+}
+
+.dom-container {
+  overflow: hidden;
+}
+
+.dom-container:hover, .sidebar:hover {
+  overflow: scroll;
```

Comment:
Have you considered `overflow: auto`? With the current `width` set to `50%`, I get a scrollbar appearing when I hover over the treeView, despite the elements only taking up like 1/4 of the real estate inside.

I'm probably reading too much into this, though. We can figure out these small adjustments near the end with a more finished product :smiley: